5.0/config/jvm.options

```diff
@@ -0,0 +1,100 @@
+## JVM configuration
```

Comment:
I'm hesitant about this file; is it necessary?  Maybe it should just be an example in the docs?

There are assumptions about available memory that may not work (esp in a default docker-toolbox VM for OSX and Windows).